content/monitors/monitor_types/composite.md

```diff
@@ -131,11 +131,10 @@ Recall the seven statuses a monitor may have (in order of increasing severity):
 * `Warn`,
 * `Alert`,
 * `Skipped`,
-* `Ignored`,
 * `No Data`,
 * `Unknown`.
 
-Composite monitors consider `Unknown`, `Warn` and `Alert` to be alert-worthy (i.e. true). The rest—`Ok`, `Skipped`, `Ignored`, and `No Data`—are not alert-worthy (i.e. false). However, you can configure `No Data` to be alert-worthy by setting `notify_no_data` to true.
+Composite monitors consider `Unknown`, `Warn` and `Alert` to be alert-worthy (i.e. true). The rest—`Ok`, `Skipped`, and `No Data`—are not alert-worthy (i.e. false). However, you can configure `No Data` to be alert-worthy by setting `notify_no_data` to true.
```

Comment:
Serial Comma:

Composite monitors consider `Unknown`, `Warn`, and `Alert` to be alert-worthy (i.e. true). The rest—`Ok`, `Skipped`, and `No Data`—are not alert-worthy (i.e. false). However, you can configure `No Data` to be alert-worthy by setting `notify_no_data` to true.